pkg/operator/revisionpoddeployer/oauthrevisionpoddeployer.go

```diff
@@ -0,0 +1,31 @@
+package revisionpoddeployer
+
+import (
+	"github.com/openshift/library-go/pkg/operator/encryption/statemachine"
+)
+
+type oauthRevisionPodDeployer struct {
+	statemachine.Deployer
+	isOAuthEncryptionConfigManagedByThisOperatorFunc func() bool
```

Comment:
just call it `enabled`. 

in other words: I think it is much clearer if these generic bits, are named in a generic way, plus the wiring of these primitives then tells the story of "AuthEncryptionConfigManagedByThisOperator" in one place.